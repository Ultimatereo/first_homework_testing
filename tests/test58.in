10
-927 737 844 20 255 -122 534 -492 892 265 893
-566 364 -36 -218 122 964 -147 -959 -411 671 239
-913 -459 377 -998 24 407 -304 451 -577 639 112
-142 -341 330 217 247 -911 123 550 -864 -767 590
-334 489 -749 394 32 -627 676 135 529 -914 -809
-46 -977 873 730 -683 460 633 -465 -613 113 11
875 422 -591 -402 -170 -996 -106 60 757 -386 -412
-950 595 -541 776 253 -83 561 -739 345 -205 981
812 407 634 -737 115 335 155 77 550 68 -976
-791 -400 -458 -181 -694 -923 -480 -718 -938 487 35
