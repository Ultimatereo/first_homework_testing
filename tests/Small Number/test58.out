-1.17037
-5.2151
0.588791
-0.919744
4.23328
-0.0979767
-2.81165
-3.90244
2.80682
-3.17228
